src/tbbbind/CMakeLists.txt

```
@@ -37,6 +37,10 @@ function(tbbbind_build TBBBIND_NAME REQUIRED_HWLOC_TARGET)
         $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
         ${HWLOC_INCLUDE_DIRS} # pkg-config defined
     )
+	target_link_directories(${TBBBIND_NAME}
+		PRIVATE
+		${HWLOC_LIBRARY_DIRS} # pkg-config defined
+	)
```

The `target_link_directories` command was introduced in CMake 3.13, but the oneTBB CMake minimal supported version is 3.1. So I would propose you check the command availability before the usage:
```suggestion
    if (COMMAND target_link_directories)
	target_link_directories(${TBBBIND_NAME}
		PRIVATE
		${HWLOC_LIBRARY_DIRS} # pkg-config defined
	)
    endif()
```
@AlexVeprev what do you think, should we have any `else` branch here to handle link directories when using CMake < 3.13?